src/vars.less

```diff
@@ -37,7 +37,7 @@
 @grid-break-wall: 90em; // ~1440px
 
 // System Font Stacks
-@systemMonospace: "SF Mono", "Monaco", "Inconsolata", "Fira Mono", "Droid Sans Mono", "Source Code Pro", monospace;
```

Comment:
`Monaco` only has the "Regular" (400) weight

`Menlo` provides "Regular", "Italic", "Bold", "Bold Italic"